src/app/components/inventory-page/inventory-page.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+import { HeroService } from 'src/app/services/heroservice/hero.service';
+import { HeroModel } from './../../models/heroModel';
+import { ActivatedRoute } from '@angular/router';
+import { Equipment } from 'src/app/models/equipment';
+
+@Component({
+  selector: 'app-inventory-page',
+  templateUrl: './inventory-page.component.html',
+  styleUrls: ['./inventory-page.component.scss'],
+})
+export class InventoryPageComponent implements OnInit {
+  private _hero: HeroModel;
+  private _id: number;
+  public _heroInventory: Equipment[];
```

Comment:
this variable is redundancy, you can just as well get it from the hero field

Reply:
true, deleted and changed